src/main/kotlin/world/cepi/sabre/server/flatgenerator/FlatLayer.kt

```diff
@@ -7,7 +7,7 @@ import net.minestom.server.instance.block.Block
 @Serializable
 data class FlatLayer(
         /** The block enum to use to represent the layer */
-        val block: Block,
+        val block: String,
```

Comment:
This should still use a Block, just during serialization add a Block serializer of sorts.